packages/Video/Video.md

```diff
@@ -21,6 +21,10 @@ Each video will appear in the "Quality" dialogue from top to bottom sorted by `q
 
 Tracks are supplied to the player through the `tracks` prop, which accepts an array of objects. Each object in the array represents a track file. Typically, these will be different languages or separate files for closed captions, subtitles, or descriptions. If no tracks are provided, the captions button will not be visible in the player.
 
+#### Analytics tracking
```

Comment:
Would it be best for the `analyticsObject` to include all possible metrics at every callback? This way we can add props to the analytics object over time and avoid breaking changes. I recommend returning a full object at every callback.

```
// Customer clicks play
analyticsObject = {
  state: 'play',
  elapsedTime: 0,
  totalTime: 60
}

// Customer clicks pause after 20 seconds
analyticsObject = {
  state: 'pause',
  elapsedTime: 20,
  totalTime: 60
}
```

Reply:
So, I have been requested by the analytics team to return an object that contains "name", "action" and "details". Name, and details we are going to handle on the application side, but action we need from the component and it has a very specific format (which is what we are returning in the component). It needs to have contain a string such as:
"action": "play", // or "pause", "watched 25%", etc.
So I am following that format, and my understanding is that this should work for all teams and it should match the analytics object that the web videos return (from YouTube)